CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Table of Contents
 
+ - [2.0.0](#200---WIP)
```

Comment:
We normally log the individual pre-releases before folding all changes into a single 2.0.0 entry at the final release, to show the specific changes in subsequent pre-release versions.
```suggestion
 - [2.0.0-alpha.1](#200-alpha1---TBD)
```